src/bucket/test/BucketIndexTests.cpp

```diff
@@ -272,6 +357,177 @@ class BucketIndexTest
         }
     }
 
+    void
+    testLoadContractKeySpanningPages()
+    {
+        size_t page_size = 16384;
```

Comment:
Instead of using a literal, you should use `cfg.EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT`